src/store/messages/saga.ts

```diff
@@ -367,6 +372,14 @@ export function* clearMessages() {
   yield put(removeAll({ schema: schema.key }));
 }
 
+export function* sendBrowserNotification(channelId, message) {
+  const channel = yield select(rawChannelSelector(channelId));
```

Comment:
This is assuming we have ALL the conversations already loaded. Is that the case? What if the conversation is past the first page of 100? What happens?